src/profiles.js

```diff
@@ -3,11 +3,10 @@ import _ from 'lodash'
 import path from 'path'
 import Utils from './utils'
 
-
 // Internal
 
-const DEFAULT_REMOTE_PATH = 'http://schemas.datapackages.org/registry.json'
-let DEFAULT_LOCAL_PATH
+const DEFAULT_REMOTE_PATH = 'https://schemas.frictionlessdata.io/registry.json'
```

Comment:
It's possible with modification of the `Utils._isRemoteURL` to include `//` and while in node this works fine, I'm getting this error when testing in the browser:

```
SyntaxError
      at XMLHttpRequest.open (node_modules/jsdom/lib/jsdom/living/xmlhttprequest.js:458:15)
      at about:blank:49369:12
      at self.fetch (about:blank:49346:13)
      at _readURL (about:blank:25508:17)
      at Function.readFileOrURL (about:blank:25534:19)
      at Profiles._loadFile (about:blank:25400:33)
      at Profiles._loadRegistry (about:blank:25286:20)
      at about:blank:25206:14
      at new Profiles (about:blank:25205:13)
      at _callee$ (test/browser/profiles.js:18:30)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:64:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:355:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:116:21)
      at step (test/browser/profiles.js:19:191)
      at test/browser/profiles.js:19:437
      at Context.<anonymous> (test/browser/profiles.js:19:99)

```